lib/vmdb/loggers.rb

```diff
@@ -59,6 +59,7 @@ def self.create_loggers
         $scvmm_log     = MirroredLogger.new(path_dir.join("scvmm.log"),      "<SCVMM> ")
         $api_log       = MirroredLogger.new(path_dir.join("api.log"),        "<API> ")
         $miq_ae_logger = MirroredLogger.new(path_dir.join("automation.log"), "<AutomationEngine> ")
+        $hawk_log      = MirroredLogger.new(File.join(path_dir, 'hawkular.log'),   '<Middleware> ')
```

Comment:
why is there a separate log for hawkular? other providers don't have their own log